Changed buttonScript and ProceduralGrid2 script

kylemas opened the pull request:
Added another handler to the buttonScript so we can toggle the visibility of the controller diagram. Changed ProceduralGrid2 script so the wave updates when paused, however this doesnt work for the gaussian wave.

willjvan committed:
Changed buttonScript and ProceduralGrid2 script

Added another handler to the buttonScript so we can toggle the visibility of the controller diagram. Changed ProceduralGrid2 script so the wave updates when paused, however this doesnt work for the gaussian wave.

kylemas commented:
@willjvan 

Hey Will!!

I was just checking out your branch and for some reason, the button wasn't toggling the controller image on the side. I reviewed the function you added in the scripts and it seemed okay to me. I was wondering if you got it working a while ago?

kylemas commented:
I was thinking you could set the image panel as a public game object so that you can drag it onto the script!

kylemas committed:
Chaned gameobject to public variable, added the image panel as the game object, added filler controller image

willjvan commented:
Hey @kylemas. It was working earlier. Maybe I changed something by accident! I noticed you made a new commit, is it working now?

willjvan committed:
Edited buttonScript, removed ability to close menu

Changed buttonScript so controlDiagram changes visibility correctly. Also removed UIToggle script from MenuCanvas so the ability to close the menu by pressing 'a' is gone
